src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const newRobot = {
+    name,
+    version,
+    wheels,
+
+    get info() {
+      return `name: ${this.name},`
+      + ` chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
```

Comment:
Follow the order:
- properties
- getters
- setters
- methods

```suggestion
    coords: {
      x: 0,
      y: 0,
    },
  
    get info() {
      return `name: ${this.name},`
      + ` chip version: ${this.version},`
      + ` wheels: ${this.wheels}`;
    },
```